src/public-lib/components/WindowWrapper.ts

```diff
@@ -10,24 +10,60 @@ export interface WindowSize {
 export default class WindowWrapper
 {
     @observable public size: WindowSize;
+    @observable public sizeByMs:{
```

Comment:
not sure what sizeByMs means.  can you spell out Ms?

Reply:
I think ms is milliseconds.   but if the value is the same, then why?

Reply:
ms is milliseconds. the value is not the same, its different debouncing amounts. definitely open to different naming structure for this